USER_MANUAL.md

```diff
@@ -866,17 +866,17 @@ It is important to note that LDAR-Sim internally converts KG of methane to MMBtu
 
 **Description:** The following parameters are used to estimate the total emission amount according to the specified program's work practices.
 
-#### &lt;duration_factor&gt; WIP
+#### &lt;duration_factor&gt;
 
 **Data Type:** Numeric
 
-**Default input:** 1
+**Default input:** 1.0
 
 **Description:** A decimal number representing the ratio of time since the last survey or screening at a given site, used to estimate the duration of a given measurement.
 
-For example, a value of 0.5 means that all measurements are based on the assumption that half the time since the last survey or screening at a given site is used to estimate the duration of the measured emission.
+For example, a value of 0.6 means that 60% of the duration was assumed to be the greater emitting value, between the two measurement points, and 40% of the duration would be assuming the smaller of the measured emission.
```

Comment:
Thoughts on including an timeseries image here as a visual resource?

Reply:
I don't think this is necessary.